quantecon/game_theory/normal_form_game.py

```diff
@@ -233,7 +240,7 @@ def delete_action(self, action, player_idx=0):
         payoff_array_new = np.delete(self.payoff_array, action, player_idx)
         return Player(payoff_array_new)
 
-    def payoff_vector(self, opponents_actions):
+    def payoff_vector(self, opponents_actions: npt.ArrayLike) -> np.ndarray:
```

Comment:
`opponents_actions` should have been `IntOrArray`, which is `Union[int, npt.ArrayLike]`.
But doing so will cause problem in https://github.com/QuantEcon/QuantEcon.py/blob/d975adb645d6879ae96f9a05ff3613322d57bef2/quantecon/game_theory/normal_form_game.py#L270. A `Union[int, ...]` type generally can't be indexed. Alternative solution to my current version is to use `cast()` before the indexing to cast `opponents_actions` into an indexable.

Note that `int` is also part of `npt.ArrayLike`, since `np.array(1)` works.